File: swea/swea_10505_소득불균형.py
T = int(input())

for test in range(T):
    n = int(input())
    arr = list(map(int, input().split()))
   
    sum_num = 0
    for i in range(n):
        sum_num += arr[i]
    mid = sum_num//n
    cnt = 0

    sorted(arr)
    for i in arr:
        if i <= mid:
            cnt += 1

    print('#%d %d'%(test+1, cnt))

File: swea/swea_10505_소득불균형.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main()
{
	int T, test_case;
	cin >> T;

	for(test_case = 1; test_case <= T; test_case++)
	{
		int N;
		cin >> N;

		int arr[10001] = {0,};
		long long sum = 0;
		long long mid = 0;
		int cnt = 0;

		for(int i = 0; i < N ; i++) 
		{
			int num = 0;
			cin >> num;
			sum += num;
			arr[i] = num;
		}

		mid = sum / N;
		sort(arr, arr+N);

		for(int i = 0; i < N; i++)
		{
			if(arr[i] > mid) break;
			else if(arr[i] <= mid) cnt++;
		}

		cout << '#' << test_case << " " << cnt << endl;

	}
	return 0;
}